pkg/devfile/parser/data/2.0.0/devfileJsonSchema200.go

```diff
@@ -1,2730 +1,872 @@
-          },
-          "github": {
-            "description": "Project's GitHub source",
-            "properties": {
-              "branch": {
-                "description": "The branch to check",
-                "type": "string"
-              },
-              "location": {
-                "description": "Project's source location address. Should be URL for git and github located projects, or; file:// for zip",
-                "type": "string"
-              },
-              "sparseCheckoutDir": {
-                "description": "Part of project to populate in the working directory.",
-                "type": "string"
-              },
-              "startPoint": {
-                "description": "The tag or commit id to reset the checked out branch to",
-                "type": "string"
-              }
-            },
-            "type": "object",
-            "additionalProperties": false
-          },
-          "name": {
-            "description": "Project name",
-            "type": "string"
-          },
-          "zip": {
-            "description": "Project's Zip source",
-            "properties": {
-              "location": {
-                "description": "Project's source location address. Should be URL for git and github located projects, or; file:// for zip",
-                "type": "string"
-              },
-              "sparseCheckoutDir": {
-                "description": "Part of project to populate in the working directory.",
-                "type": "string"
-              }
-            },
-            "type": "object",
-            "additionalProperties": false
-          }
-        },
-        "required": [
-          "name"
-        ],
-        "type": "object",
-        "additionalProperties": false,
-        "oneOf": [
-          {
-            "required": [
-              "git"
-            ]
-          },
-          {
-            "required": [
-              "github"
-            ]
-          },
-          {
-            "required": [
-              "zip"
-            ]
-          }
-        ]
-      },
-      "type": "array"
-    },
-    "metadata": {
-      "type": "object",
-      "description": "Optional metadata",
-      "properties": {
-        "version": {
-          "type": "string",
-          "description": "Optional semver-compatible version",
-          "pattern": "^([0-9]+)\\.([0-9]+)\\.([0-9]+)(\\-[0-9a-z-]+(\\.[0-9a-z-]+)*)?(\\+[0-9A-Za-z-]+(\\.[0-9A-Za-z-]+)*)?$"
-        },
-        "name": {
-          "type": "string",
-          "description": "Optional devfile name"
-        }
-      }
-    },
-    "schemaVersion": {
-      "type": "string",
-      "description": "Devfile schema version",
-      "pattern": "^([2-9]+)\\.([0-9]+)\\.([0-9]+)(\\-[0-9a-z-]+(\\.[0-9a-z-]+)*)?(\\+[0-9A-Za-z-]+(\\.[0-9A-Za-z-]+)*)?$"
-    }
-  },
-  "type": "object",
-  "additionalProperties": false,
-  "required": [
-    "schemaVersion"
-  ]
-}`
+	"description": "Devfile schema.",
```

Comment:
This doesn't look right. The original schema had a thousands of lines, now it is hundreds

Reply:
I agree! Will take a look

Reply:
Definitely not right :-(
Think we need to update this to be the larger one based off the  one in the `master` branch